cmd/lncli/commands.go

```diff
@@ -16,6 +16,8 @@ import (
 	"sync"
 	"syscall"
 
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
```

Comment:
Lack of commit message body for such a significant change.

Reply:
This PR wasn't ready for review yet, commit messages not final.